src/test/scala/chiselTests/util/SRAMSpec.scala

```diff
@@ -100,7 +100,7 @@ class SRAMSpec extends ChiselFlatSpec {
       val fooIo = IO(foo.cloneType)
       fooIo :<>= foo
     }
-    val chirrtl = emitCHIRRTL(new Top)
+    val chirrtl = emitCHIRRTL(new Top, Array("--full-stacktrace"))
```

Comment:
```suggestion
    val chirrtl = emitCHIRRTL(new Top)
```
No reason to commit that, that's just for debugging.